OpenRA.Game/Sound/Sound.cs

```diff
@@ -113,6 +113,16 @@ public void StopAudio()
 			soundEngine.StopAllSounds();
 		}
 
+		public void MuteAudio()
+		{
+			soundEngine.Volume = 0f;
+		}
+
+		public void UnmuteAudio()
+		{
+			soundEngine.Volume = 1f;
```

Comment:
You should probably be setting this to whatever the player has set, not necessarily full volume.